articles/multifactor-authentication/mfa-from-id-token.md

```diff
@@ -0,0 +1,28 @@
+---
+description: Describes how to determine if a user has done multifactor authentication using their id_token and JWT
+---
+
+## Determine if a user has performed multifactor authentication
+
+At times, it's necessary to determine if a particular user has had the additional security of multifactor authentication applied to their session. For instance, a user may be allowed access to sensitive data or allowed to reset their password only after further confirming their identity using MFA.
+
+To determine this for a particular user session, developers can check the claims information available on the `id_token`. 
+The `id_token` contains claims information relevant to the usage of multifactor authentication. After retrieving the `id_token`, [as shown here](tokens/id_token), the value of the `amr` claim can be evaluated.
+
+```
```

Comment:
I think you could specify the language (js) after the \`\`\` so you get better highlighting. I'd also mention that the example is based on https://github.com/auth0/node-jsonwebtoken and maybe you could create a `const` on top with the secret, something like:

```
const AUTH0_CLIENT_SECRET = '${account.clientSecret}'
```

that'd be replaced on runtime https://github.com/auth0/docs/blob/master/CONTRIBUTING.md#document-variables